db/migrate/20230202105819_add_confirmable.rb

```diff
@@ -0,0 +1,11 @@
+class AddConfirmable < ActiveRecord::Migration[7.0]
+  def change
+    change_table(:members) do |t| 
+      t.string   :confirmation_token
+       t.datetime :confirmed_at
+       t.datetime :confirmation_sent_at
+       t.string   :unconfirmed_email 
+    end
+    add_index  :members, :confirmation_token, :unique => true 
+  end
```

Comment:
- [ ] When I try to run the migration, I get the following error in the following image, kindly fix this issue.
> ![image](https://user-images.githubusercontent.com/53402600/216703295-59874e31-bccb-4fc8-85c3-3495cc908b08.png)

Reply:
> * [ ]  When I try to run the migration, I get the following error in the following image, kindly fix this issue.
> 
> > ![image](https://user-images.githubusercontent.com/53402600/216703295-59874e31-bccb-4fc8-85c3-3495cc908b08.png)

Hi @omarsalem7,
Find attached a video demonstrating the required features
https://www.loom.com/share/2f9abb4a39cd4ab7bc39b7795b15d889